web_src/js/utils/match.ts

```diff
@@ -41,3 +44,15 @@ export function matchMention(queryText: string): string[] {
 
   return sortAndReduce(results);
 }
+
+export async function matchIssue(owner: string, repo: string, _issueIndex: string, queryText: string): Promise<Issue[]> {
```

Comment:
Why underscore `_` here ....

Reply:
The same variable name was used below to convert to a number. In some projects it's pretty common to prefix such vars with a leading `_`. However renamed it here.